me-lazyload.js

```diff
@@ -99,6 +100,19 @@ angular.module('me-lazyload', [])
 
             iElement.bind('load', onLoad);
 
+            var $scrollContainer = $win
+            $scrollContainer.on('scroll', checkImage);
+
+
+            $scope.$watch('scrollContainer', function(newVal){
+                if(newVal){
+                    $scrollContainer.off('scroll', checkImage);
+                    $scrollContainer = angular.element(document).find($scope.scrollContainer);
+                    $scrollContainer.on('scroll', checkImage);
```

Comment:
Everytime a new lazy-src directive created, the scrollContainer will add a event listener? So, the scrollContainer will have many events?